tests_metricflow/generate_snapshots.py

```diff
@@ -142,6 +143,10 @@ def run_tests(test_configuration: MetricFlowTestConfiguration) -> None:  # noqa:
     if test_configuration.engine is SqlEngine.DUCKDB:
         # DuckDB is fast, so generate all snapshots, including the engine-agnostic ones
         run_command(f"pytest -x -vv -n 4 --overwrite-snapshots -k 'not itest' {TEST_DIRECTORY}")
+
+        # Run snapshots changes for metricflow-semantics
+        # these are not dialect specific, so only need to run once
+        run_command(f"pytest -x -vv -n 4 --overwrite-snapshots {MF_SEMANTICS_TEST_DIRECTORY}")
```

Comment:
put this to be ran under the `duckdb` engine only, as we only need to run these updates once since it's not dialect specific

Reply:
Did you test this to make sure it works as expected?

Reply:
yup!